confluence-xml/src/test/resources/confluencexml/misc2.test

```diff
@@ -0,0 +1,2954 @@
+.#------------------------------------------------------------------------------
```

Comment:
Would be better to have a task dedicated test case which reflect in the test file name and which does not contain other things.